api/URLSearchParams.json

```diff
@@ -353,6 +393,7 @@
       "forEach": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/URLSearchParams/forEach",
+          "spec_url": "https://url.spec.whatwg.org/#dom-urlsearchparams-urlsearchparams",
```

Comment:
FYI, also added spec link to top level for each of the items added by the iterator in the IDL (because these aren't expanded out in spec.